dedalo/dedalo_users_auth.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
```

Comment:
Please use `/bin/bash` and add the license header.
Header example: https://docs.nethserver.org/projects/nethserver-devel/en/latest/license_headers.html (correct the year after copying and pasting)